website/templates/emails/welcome_osf4i.html.mako

```diff
@@ -11,37 +11,50 @@
 
 Hello ${user.fullname},<br>
 <br>
-Thank you for verifying your OSF account, a free, open source service maintained by the Center for Open Science. Here are a few things you can do with the OSF:<br>
+Thank you for verifying your on OSF, a free, open source service maintained by the Center for Open Science. Here are a few things you can do with the OSF:<br>
 <br>
+
+<h4>Select your storage location</h4>
+Files can be stored in a location you specify from the available geographic regions for new projects. <a href="https://osf.io/settings/account/?utm_source=notification&utm_medium=email&utm_campaign=welcome">Set storage location.</a><br>
+<br>
+
 <h4>Store your files</h4>
-Archive your materials, data, manuscripts, or anything else associated with your research during the research process or after it is complete.<br>
+Archive your materials, data, manuscripts, or anything else associated with your work research during the research process or after it is complete. <a href="http://help.osf.io/m/projectfiles/l/482002-upload-files/?utm_source=notification&utm_medium=email&utm_campaign=welcome">Learn how.</a><br>
```

Comment:
"Archive your materials, data, manuscripts, or anything else associated with your work research"
should be 
"Archive your materials, data, manuscripts, or anything else associated with your work"